danielMunteanu/src/main/java/danielCollectionTask/Company.java

```diff
@@ -0,0 +1,23 @@
+package danielCollectionTask;
+
+import java.util.List;
+
+public class Company extends AbstractOrder {
+    private Company company;
+
+    public Company(List<Item> items, Company company) {
+        super(items);
+        this.company = company;
+    }
+
+    public Company getCompany() {
+        return company;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
this is an automatically generated toString method, could you please make it custom by adding some parent properties?

Reply:
done